normandy/recipes/tests/test_api.py

```diff
@@ -249,3 +252,25 @@ def test_unavailable_if_admin_disabled(self, api_client, settings):
         res = api_client.get('/api/v1/recipe/')
         assert res.status_code == 403
         assert res.data['detail'] == AdminEnabled.message
+
+
+def test_version(client, mocker):
```

Comment:
This test (and `test_lbheartbeat_makes_no_db_queries`) were in `normandy/health`, and probably got duplicated.